src/SmartComponents/BaselinesTable/redux/helpers.js

```diff
@@ -1,5 +1,22 @@
 import moment from 'moment';
 
+function fetchBaselines (tableType, fetchBaselines, fetchParams = {}) {
+    let params = {};
+
+    if (Object.entries(fetchParams).length !== 0) {
+        /*eslint-disable camelcase*/
+        params.order_by = fetchParams.orderBy;
```

Comment:
`.orderBy` could be undefined on the fetchParams object here. Same is true for any of the properties.